utils.js

```diff
@@ -400,6 +406,31 @@ const createMethods = async ({ apig, apiId, endpoints }) => {
   return endpoints
 }
 
+const createModel = async ({ apig, apiId, model }) => {
+  const params = {
+    'contentType': 'application/json',
```

Comment:
Is there any case where we wouldn't want this to be hardcoded? I gues if we were doing file uploads we wouldn't be using models anyways ...